applications/serial_lte_modem/src/main.c

```diff
@@ -206,30 +269,29 @@ static void gpio_cb_func(const struct device *dev, struct gpio_callback *gpio_ca
 	}
 	err = slm_at_host_power_on();
 	if (err) {
-		(void)atomic_set(&gpio_cb_running, false);
+		atomic_set(&callback_running, false);
 		LOG_ERR("Failed to power on uart: %d", err);
 		return;
 	}
 
-	gpio_pin_interrupt_configure(gpio_dev, CONFIG_SLM_WAKEUP_PIN, GPIO_INT_DISABLE);
-	gpio_remove_callback(gpio_dev, gpio_callback);
-	(void)atomic_set(&gpio_cb_running, false);
+	gpio_remove_callback(dev, gpio_callback);
+
+	/* Enable the poweroff interrupt only when the pin will be back to a nonactive state. */
+	err = configure_power_pin_interrupt(power_pin_callback_enable_poweroff,
+					    GPIO_INT_EDGE_RISING);
+
+	atomic_set(&callback_running, false);
 }
 
-void enter_idle(void)
+void slm_enter_idle(void)
```

Comment:
Idle is still consuming 10uA, instead of < 3uA.

Reply:
Thanks for testing this out. We should make sure that this issue is known about.

Reply:
I don't think that this is necessarily a bug, as https://docs.zephyrproject.org/latest/hardware/peripherals/gpio.html is a bit unclear with this. But I will create a bug about the increased power consumption with edge triggering.

Reply:
Yeah, I was pointing at the increased power consumption.